docs/src/layouts/GuidelinesLayout/GuidelinesLayout.js

```diff
@@ -23,7 +23,7 @@ import SideNavLayout from '../SideNavLayout';
 import links from '../links';
 
 const StyleGuideLayout = ({ children, ...rest }) => (
```

Comment:
Does this need to be renamed to `GuidelinesLayout`?

Reply:
Yeo, good catch :)